docs/secure-connections/sauce-connect-5/installation/docker.md

```diff
@@ -20,11 +20,11 @@ Here are some benefits/use cases for using containerized Sauce Connect Proxy:
 ## Running the Sauce Connect Docker Container
 
 1. Pull the Sauce Connect Proxy Docker Image from the [Docker Hub](https://hub.docker.com/r/saucelabs/sauce-connect).
-   - To pull the latest version of Sauce Connect Proxy (recommended):
+    - To pull the latest version of Sauce Connect Proxy (recommended):
```

Comment:
just noticed:
```
To pull the latest patch version of Sauce Connect Proxy 5.0
```

Reply:
The docker instructions will be completely reworked.